### DISCUSSION TITLE
Uninstall-PSResource -WhatIf doesn't work?

### DISCUSSION THREAD

**Discussion by Viajaz** [PowerShell/PSResourceGet#875] [Q&A]
I ran a script with `Uninstall-PSResource` and  `-WhatIf` and it seemed to actually uninstall the modules instead. I don't have time to develop a reproduction but has anyone else experienced this?

**Comment by jdhitsolutions** [comment]
I don't see this behavior in 3.0.19-beta19. You mention running `Uninstall-PSResource` in a script. It is possible your script isn't properly passing preferences to the PowerShellGet commands.
